packages/features/bookings/Booker/store.ts

```diff
@@ -25,6 +25,7 @@ type StoreInitializeType = {
   verifiedEmail?: string | null;
   rescheduleUid?: string | null;
   seatReferenceUid?: string;
+  durationConfig?: number[] | null;
```

Comment:
We now accept a duration configuration to enable calculation of allowed durations